web/src/user_group_popover.ts

```diff
@@ -99,13 +107,14 @@ export function toggle_user_group_info_popover(element, message_id) {
     );
 }
 
-export function register_click_handlers() {
+export function register_click_handlers(): void {
     $("#main_div").on("click", ".user-group-mention", (e) => {
         e.stopPropagation();
 
         const $elt = $(e.currentTarget);
         const $row = $elt.closest(".message_row");
-        const message = message_lists.current.get(rows.id($row));
+        assert(message_lists.current);
```

Comment:
Never write ambiguous truthiness checks that make it unclear which falsy value you’re guarding against.

```suggestion
        assert(message_lists.current !== undefined);
```